applications/forms.py

```diff
@@ -236,8 +236,11 @@ def clean(self):
             raise forms.ValidationError("Please enter a school from suggested.")
 
         uniemail = cleaned_data.get('uniemail')
+        edu_verifier = '.edu' not in uniemail #and '.ca' not in uniemail
```

Comment:
Would recommend keeping the and '.ca' part since if we do end up doing physical in the future we'll prob still do US + CA